docs/proposals/19-migrating-to-prometheus-operator.md.md

```diff
@@ -452,43 +452,69 @@ out in the following steps:
 5. Add a [sidecar][grafana-sidecar] to grafana that will pickup dashboards and
    provision them. Each dashboard gets its own configmap.
 
-    - Most dashboards provisioned by Grafana are the same for each shoot
-      cluster. To avoid unnecessary duplication of configmaps, the dashboards
-      could be added once in a single namespace. These "common" dashboards can
-      then be discovered by each Grafana and provisioned.
+    - Grafana in the control plane
 
-    - In some cases, dashboards are more "specific" because they are related to
-      a certain Kubernetes version.
+      - Most dashboards provisioned by Grafana are the same for each shoot
+        cluster. To avoid unnecessary duplication of configmaps, the dashboards
+        could be added once in a single namespace. These "common" dashboards can
+        then be discovered by each Grafana and provisioned.
 
-    - Contract between dashboards in configmaps and the Grafana sidecar.
+      - In some cases, dashboards are more "specific" because they are related to
+        a certain Kubernetes version.
 
-      - Each common dashboard will be deployed in the `monitoring` namespace as
-        a configmap. The configmap will be labeled with
-        `gardener.cloud/role=dashboard`.
+      - Contract between dashboards in configmaps and the Grafana sidecar.
 
-      - Each specific dashboard will be deployed in the shoot namespace. The
-        configmap will also be labeled with `gardener.cloud/role=dashboard`.
+        - Each common dashboard will be deployed in the `monitoring` namespace as
+          a configmap. The configmap will be labeled with
+          `gardener.cloud/dashboard=shoot`.
```

Comment:
Shall we use `gardener.cloud/dashboard-target=shoot` for symmetry with the prometheus target?